app/controllers/api_controller.rb

```diff
@@ -122,11 +122,11 @@ def set_access_control_headers
   end
 
   def base_config
-    Api::Settings.base
+    ManageIQ::API::Settings.base
```

Comment:
@imtayadeway I'm going to merge anyway, that was not in the scope of the PR.  I'm just curious.